libcodechecker/log/build_action.py

```diff
@@ -12,6 +12,11 @@
 
 
 class BuildAction(object):
+    LINK = 0
+    COMPILE = 1
+    PREPROCESS = 2
```

Comment:
Can't we have build actions that do multiple things? I.e. both compile and link? Is it sufficient to have these constants? Shouldn't they rather be powers of two so we can have a set of purpose for each build action?

Reply:
We could make it an optional dependency. If we use the tool from CodeChecker using the API we could have skip_handler as an optional parameter, but it would not be exposed when used from command line.